<?xml version="1.0"?>
<launch>
    
    <!--enable_scanning-->
    <arg name="enable_task" default="true"/>

    <arg name="initial_waypoint_id" default="1"/>

<!--    <arg name="odometry_topic" default="/odom_gps"/>-->
    <arg name="odometry_topic" default="/ekf_slam_node/slam_odom"/>

    <!-- <node name="mid_cont" pkg="sherpa_ros" type="sherpa_mid_cont_node"/> -->
    <node name="map_manager" pkg="sherpa_ros" type="map_manager_node"/>

    <node name="task_manager" pkg="sherpa_ros" type="task_manager.py" output="screen">
        <param name="reference" value="geo" />
        <param name="task_name" value="scanning" />
	    <param name="tour" value="current_mission/tour" />
        <param name="map_points" value="current_mission/map" />
        <param name="odometry_topic" value="$(arg odometry_topic)" />
        <param name="enable_task" value="$(arg enable_task)" />
        <param name="threshold" value="0.25"/>
        <param name="threshold_waypoint" value="0.30"/>
        <param name="initial_waypoint_id" value="$(arg initial_waypoint_id)"/>
    </node>


</launch>
